content/id/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,459 @@
+dengan metrik yang dispesifikasikan pada HorizontalPodAutoscaler. Kontroler manajer mendapat
+metrik dari *resource* metrik API (untuk metrik per *pod*) atau dari *custom* metrik API (untuk semua metrik lainnya).
+
+During each period, the controller manager queries the resource utilization against the
+metrics specified in each HorizontalPodAutoscaler definition.  The controller manager
+obtains the metrics from either the resource metrics API (for per-pod resource metrics),
+or the custom metrics API (for all other metrics).
+
+* Untuk metrik per *pod* (seperti CPU), kontroler mengambil metrik dari *resource* metrik API
+  untuk setiap *pod* yang ditargetkan oleh HorizontalPodAutoscaler. Kemudian, jika nilai target penggunaan ditentukan,
+  maka kontroler akan menghitung nilai penggunaan sebagai persentasi dari pengguaan *resource* dari kontainer 
+  pada masing-masing *pod*. Jika target *raw value* ditentukan, maka nilai *raw metric* akan digunakan
+  secara langsung. Kontroller kemudian mengambil nilai rata-rata penggunaan atau *raw values* (tergantung
+  dengan tipe target yang ditentukan) dari semua *pod* yang targetkan dan menghasilkan perbandingan yang
+  digunakan untuk menentukan jumlah replika yang akan di-*scale*.
+
+  Perlu dicatat bahwa jika beberapa kontainer pada pod tidak memiliki nilai *resource request*, penggunaan CPU
+  pada pod tersebut tidak akan ditentukan dan *autoscaler* tidak akan melakukan tindakan apapun untuk metrik tersebut.
+  Perhatikan pada bagian [detail algoritma](#algorithm-details) dibawah ini untuk informasi lebih lanjut mengenai
+  cara kerja algorihma *autoscale*.
+
+* Untuk *custom* metrik per *pod*, kontroler bekerja sama seperti *resource* metrik per *pod*,
+  kecuali *pod* bekerja dengan *raw values*, bukan dengan *utilization values*.
+
+* Untuk objek metrik dan eksternal metrik, sebuah metrik diambil, dimana metrik tersebut menggambarkan
+  object tersebut. Metrik ini dibandingkan dengan nilai target untuk menghasilkan perbandingan seperti diatas.
+  Pada API `autoscaling/v2beta2`, nilai perbandingan dapat secara opsional dibagi dengan jumlah *pod*
+  sebelum perbandingan dibuat. 
+
+Pada normalnya, HorizontalPodAutoscaler mengambil metrik dari serangkaian APIs yang sudah diagregat 
+(`custom.metric.k8s.io`, dan `external.metrics.k8s.io`). API `metrics.k8s.io` biasanya disediakan oleh
+*metric-server*, dimana *metric-server* dijalanjkan secara terpisah. Perhatian 
+[*metrics-server*](/docs/tasks/debug-application-cluster/resource-metrics-pipeline/#metrics-server) sebagai petunjuk.
+HorizontalPodAutoscaler juga mengambil metrik dari Heapster secara langsung.
+
+{{< note >}}
+{{< feature-state state="deprecated" for_k8s_version="v1.11" >}}
+Fetching metrics from Heapster is deprecated as of Kubernetes 1.11.
+{{< /note >}}
+
+Perhatikan [Support for metrics APIs](#support-for-metrics-apis) untuk lebih detail.
```

Comment:
which header does this refer to? make sure the the anchor link here is referring to that header.
```suggestion
Perhatikan [Dukungan untuk API metrik](#dukungan-untuk-api-metrik) untuk lebih detail.
```